content/800-data-platform/200-pulse/100-what-is-pulse.mdx

```diff
@@ -14,7 +14,7 @@ toc: true
 
 <Admonition type="warning">
 
-Pulse is currently in [Early Access](/data-platform/classic-projects/about/releases#early-access). Although we already have high confidence in it, the nature of an Early Access product is that significant iterations might happen at any time. Therefore, we advise against using it in a system that requires stability.<br /><br />
+Pulse is currently in [Early Access](/docs/data-platform/platform-console/maturity-levels#early-access). Although we already have high confidence in it, the nature of an Early Access product is that significant iterations might happen at any time. Therefore, we advise against using it in a system that requires stability.<br /><br />
```

Comment:
```suggestion
Pulse is currently in [Early Access](/data-platform/platform-console/maturity-levels#early-access). Although we already have high confidence in it, the nature of an Early Access product is that significant iterations might happen at any time. Therefore, we advise against using it in a system that requires stability.<br /><br />
```

Reply:
Is there something I need to do to approve the change? 

I'm not sure why the CI is failing on the PR.

Reply:
Hey there @shahzeb1 👋 , I'm looking into it.